### DISCUSSION TITLE
Multiple errors with 2.7.0: no download, blogs removed long ago reappear and cannot be removed

### DISCUSSION THREAD

**Discussion by vozmor** [TumblThreeApp/TumblThree#277] [General]
Problems immediately started with the automatic upgrade to 2.7.0.324. So I downloaded and reinstalled the latest version manually, just in case, but the errors remained.

1) Download does not start after clicking to Donwload button. After clicking Stop button Download button remains gray instead of turning back to black. Adding to and removing from queue is all right, by both the buttons and the keyboard.

2) Dozens of blogs that were removed months ago by me now reappeared. Some of these even don't exist anymore on Tumblr, yet they are not either coloured red or labelled Offline. And these blogs cannot be removed at all ("Error: Cold not remove the blog: ..."). All the other blogs can still be removed and added.

3) I switched back to 2.6.3 and everything changed back to normal.

**Comment by thomas694** [comment]
1) Are you sure the app was already loaded completely? Some start tasks can take a couple of 'seconds'. How long after pressing Download did you wait before deciding that it isn't working? And when waiting for stop?

2) Normally for removed blogs the blog index files are deleted, with the setting "archive the index files" they are moved to a subfolder "Archive". You said you see both types of blogs, normal ones and removed/archived ones, but they should be in different folders. As long as the blogs are not used again, they won't change color or be labelled as offline (except with "check status on startup"). Is your "download location" still pointing to your usual folder? Can you confirm in windows explorer that the blog files are still separated in Index and Archive folders or are they altogether in one?

3) Are you using anything special? Do you use collections?
In general you can confirm that logging is switched on and check for any errors in the %LOCALAPPDATA%\TumblThree\TumblThree.log. So after each event (startup, download, stop) look at the end of the file, if there's a hint or error.

For starters you could also rename your settings.json and start TumblThree (and set your custom download location again if necessary) to see if it works normally again.
